src/onnx/parse_matmul.cpp

```diff
@@ -173,12 +333,20 @@ struct parse_matmul : op_parser<parse_matmul>
         }
 
         auto is_quant_dot = opd.op_name == "quant_dot";
+        auto has_scales   = opd.op_name == "quant_dot_scaled";
```

Comment:
A little confusing naming convention. Between `quant_dots` and `Matmul`**. And then there is `has_scales`: which is presumably also a `quant_dot`.

Reply:
What do you suggest I name it? quant_dot_dequant? This operator essentially takes in quantized input and dequantizes the output.

Reply:
Okay renamed it to is_quant_dot_scaled